gloo/docs/gloo_routing/virtual_services/security/apikey_auth/test-no-auth-vs.yaml

```diff
@@ -1,10 +1,9 @@
 apiVersion: gateway.solo.io/v1
 kind: VirtualService
 metadata:
-  name: test-no-auth
```

Comment:
i don't think we should rename this. this is the virtual service that doesn't have auth. the new name implies to me that this is a secured virtual service

Reply:
I worded the guide(s) so that you start with a virtual service with no auth and then add auth to it (before we were creating two distinct virtual services). I agree that the name seems to imply that the VS is secure, I can rename it to something like `auth-tutorial`. Would that work for you?